cmd/editor_config.go

```
@@ -0,0 +1,137 @@
+}
+
+// initializeConfig breaks the initialization cycle by separating the config setup
+func initializeConfig() {
+	if configFilePath == "" {
+		configFilePath = defaultConfigFilePath
+	}
+
+	var err error
+	currentConfig, err = config.NewConfig(configFilePath)
+	if err != nil {
+		logger.Error(err.Error())
```

golangci-lint (1.62.2)

printf: non-constant format string in call to github.com/editorconfig-checker/editorconfig-checker/v3/pkg/logger.Error

(govet)